docs/develop/run-code-on-document-open.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Run code in your Office Add-in when the document opens
+description: 'Learn how to run code in your Office Add-in add-in when the document opens.'
+ms.date: 12/28/2020
+localization_priority: Normal
+---
+
+# Run code in your Office Add-in when the document opens
+
+[!include[Shared JavaScript runtime requirements](../includes/shared-runtime-requirements-note.md)]
+
+You can configure your Office Add-in to load and run code as soon as the document is opened. This is useful if you need to register event handlers, pre-load data for the task pane, synchronize UI, or perform other tasks before the add-in is visible.
+
+[!include[Shared runtime note](../includes/note-requires-shared-runtime.md)]
+
+## Configure your add-in to load when the document opens
+
+The following code configures your add-in to load and start running when the document is opened.
+
+```JavaScript
+Office.addin.setStartupBehavior(Office.StartupBehavior.load);
+```
+
+> [!NOTE]
+> The `setStartupBehavior` method is asynchronous.
+
+## Configure your add-in for no load behavior on document open
+
+The following code configures your add-in not to start when the document is opened. Instead it will start when the user engages it in some way (such as choosing a ribbon button, or opening the task pane.)
```

Comment:
```suggestion
The following code configures your add-in not to start when the document is opened. Instead, it will start when the user engages it in some way, such as choosing a ribbon button or opening the task pane.
```